src/bin/tools.ts

```diff
@@ -0,0 +1 @@
+exports.server_url = "http://localhost:5000";
```

Comment:
Perhaps `config.ts` or `globals.ts` is a better name for this file. Additionally, if these are constant across all files, let's uppercase them (`SERVER_URL`).

The port will depend on the environment, so this string should probably be constructed using a `PORT` environment variable.